src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static Random random = new Random();
+    protected static Circle circle = new Circle(random.nextInt(), ColorSupplier.getRandomColor());
+    protected static IsoscelesTrapezoid isoscelesTrapezoid
+            = new IsoscelesTrapezoid(random.nextInt(),
+            random.nextInt(), random.nextInt());
+    protected static Rectangle rectangle = new Rectangle(random.nextInt(),
+            random.nextInt());
+    protected static RightTriangle rightTriangle = new RightTriangle(random.nextInt(),
+            random.nextInt());
+    protected static Square square = new Square(random.nextInt());
+
+    private static Circle getRandomCircle() {
+        return circle;
+    }
+
+    private static IsoscelesTrapezoid getRandomTrapezoid() {
+        return isoscelesTrapezoid;
+    }
+
+    private static Rectangle getRandomRectangle() {
+        return rectangle;
+    }
+
+    private static RightTriangle getRandomRightRectangle() {
+        return rightTriangle;
+    }
+
+    private static Square getRandomSquare() {
+        return square;
+    }
+
+    public FigureForm getRandomFigure(int randomFigure) {
+        randomFigure = random.nextInt(5);
```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
    public FigureForm getRandomFigure() {
        randomFigure = random.nextInt(FIGURE_COUNT);
```

Reply:
not fixed